app/views/registrations/add.html.erb

```diff
@@ -3,6 +3,10 @@
 <%= render layout: "nav" do %>
   <% if @competition.registration_full? %>
     <%= alert :warning, t('registrations.mailer.deleted.causes.registrations_full'), note: true %>
+  <% elsif !@competition.registration_currently_open? && !@competition.on_the_spot_registration? %>
+    <%= alert :warning, t('registrations.add.ots_not_enabled') %>
+  <% elsif @competition.probably_over? %>
+    <%= alert :warning, t('registrations.add.competition_over') %>
```

Comment:
About the two `alert`s: The other existing alert above has `note: true`, what does that do and is it useful here as well?

Reply:
Thans for finding out what it does. What about the other part of my question:

> and is it useful here as well?

I'm genuinely curious to hear what you think, because I would prefer to stay consistent with the alerts that we already have

Reply:
I don't think it is particularly useful - and if we want consistency, I'd suggest removing it from the existing warning here. "Note" is used to indicate some kind of additional information that might be of interest[1]. The fact that we are not rendering the OTS panel and displaying this message instead makes this alert the **main message** - so prefixing it with "Note" doesn't make sense in my view.

[1] "Note: You can still register, but you might end up on the waiting list"

Reply:
Agreed, feel free to remove it from the other messages for consistency.

Reply:
Removed